bitnami/rabbitmq/values.yaml

```diff
@@ -551,11 +575,11 @@ resources:
   ##    cpu: 1000m
   ##    memory: 2Gi
   ##
-  limits: {}
+  limits:
   ## Examples:
   ## requests:
   ##    cpu: 1000m
-  ##    memory: 2Gi
+    memory: 1Gi
```

Comment:
The current fork explains _why_ we are putting a default value in the chart (which is pretty unusual for the resources, since sizing is really specific per cluster & env).
cf: https://github.com/blablacar/helm-charts/commit/e1ddaf718be6e061918975ef239ebf779b69bfa4#diff-e906d8b6e0e0b38705f9cbcb021b105d83e10d7efbcb9959bb4be1929f55fa56R409

Why not keeping that comment ?